package.json

```diff
@@ -4,20 +4,21 @@
   "license": "MIT",
   "scripts": {
     "build": "npm run clean && mkdirp build && webpack --progress --colors",
+    "build:esmodules": "node ./scripts/build/esmodule-builder.js",
     "build:demos": "npm run clean && mkdirp build && webpack --config=demos/webpack.config.js --progress --colors",
     "build:min": "mkdirp build && cross-env MDC_ENV=production webpack -p --progress --colors",
     "changelog": "standard-changelog -i CHANGELOG.md -k packages/material-components-web/package.json",
-    "clean": "del-cli build/** build .typescript-tmp/** .typescript-tmp",
+    "clean": "del-cli build/** build .typescript-tmp/** .typescript-tmp && git clean -fX",
```

Comment:
Please don't use `git clean -fX`.

I have a bunch of log data from travis-ci.org (before we migrated to .com) sitting in directories that are manually .gitignore'd on my machine. I also have a bunch of custom TS migration and test scripts too. This command would nuke all of those files, without asking, and it's not possible to recover the log data since Travis no longer has it.

Arguably I shouldn't be keeping important stuff in a gitignore'd directory (and this is a good opportunity for me to move it elsewhere), but `git clean -fX` is still a really dangerous command that should only be used with great care.

It might be better to output generated files (like `.d.ts` and `.js`) in a subdirectory of each package, such as `dist/`, so that we can be more surgical with `clean`.

Reply:
oh no! I hope I didn't just delete all your files. It was just the easiest command to use...the del-cli package doesn't respect node-glob patterns. So it is going to be very verbose.

Reply:
I guess the alternative is add something under `scripts` ourselves?